packages/components/rollup.config.mjs

```diff
@@ -128,12 +128,14 @@ export default {
     {
       dir: "dist",
       entryFileNames: "[name].cjs",
+      chunkFileNames: "[name]-[format].js",
       exports: "named",
       format: "cjs",
     },
     {
       dir: "dist",
       entryFileNames: "[name].mjs",
+      chunkFileNames: "[name]-[format].js",
```

Comment:
I don't think there is any change since it is being bundled. It may also help with some of the vite not picking up changes when atlantis is updated 🤔